src/windows-emulator/syscalls.cpp

```diff
@@ -426,6 +448,21 @@ namespace syscalls
         return STATUS_NOT_SUPPORTED;
     }
 
+    NTSTATUS handle_NtRemoveIoCompletion(
+        const syscall_context&, const emulator_object<handle> /*io_completion__handle*/,
+        const emulator_object<int64_t> key_context, const emulator_pointer /*apc_context*/,
+        const emulator_object<IO_STATUS_BLOCK<EmulatorTraits<Emu64>>> /*io_status_block*/,
+        const emulator_object<LARGE_INTEGER> timeout)
+    {
+        if (timeout.value() && timeout.read().QuadPart == 0)
```

Comment:
can use operator bool here:
```suggestion
        if (timeout && timeout.read().QuadPart == 0)
```